src/Symfony/Component/HttpFoundation/File/File.php

```diff
@@ -104,6 +104,11 @@ public function move(string $directory, string $name = null)
         return $target;
     }
 
+    public function getContent(): string
+    {
+        return file_get_contents($this->getPathname());
```

Comment:
this will implicitly cast false to an empty string, should we throw instead?

Reply:
Good catch, fixed :+1: